### PR TITLE
Ballistics - Remove dispersion changes

### PR DESCRIPTION
**Pull request by Laid3acK**
According with https://github.com/acemod/ACE3/pull/7532#issuecomment-589994082

### CODE REVIEW

**Comment by Laid3acK** [comment]
Next `Rifle_Base_F` clean up.

File: addons/ballistics/CfgWeapons.hpp [hunk 1]
```diff
@@ -8,11 +8,98 @@ class CfgWeapons {
...
+    // Rifle_Long_Base Sniper Marksman
     // GM6 Lynx
-    class GM6_base_F: Rifle_Long_Base_F {};
+    class GM6_base_F: Rifle_Long_Base_F {
+        ACE_barrelLength = 730;
+        ACE_barrelTwist = 381.0;
+        initSpeed = -1.0;
+        magazineWell[] = {
+            "CBA_50BMG_M107"
+        }; // empty in vanilla
```
Comment (dedmen) [inline addons/ballistics/CfgWeapons.hpp]: doesn't mean that other mods aren't adding to it. += is safer

**Comment by Laid3acK** [comment]
Except if i mistaken somewhere, should fix this issue https://github.com/acemod/ACE3/issues/7559

**Comment by commy2** [comment]
If it does, it does so poorly. Still using magazines[] instead of magwells. So I'd rather keep that issue open until a proper fix.

**Pull request merged**